site/app/templates/autograding/AutoResults.twig

```diff
@@ -64,15 +64,15 @@
 
                             {# Details button #}
                             {% if testcase.has_extra_results and can_view_details %}
-                                <span class="loading-tools" id="details_tc_{{ loop.index0 }}" style="float:right">
+                            <span class="loading-tools" id="details_tc_{{ loop.index0 }}" style="float:right">
                                 <span class="loading-tools-hide" style="color: var(--standard-deep-blue); text-decoration: underline;" hidden>
                                     Hide Details
                                 </span>
                                 <span class="loading-tools-show" style="color: var(--standard-deep-blue); text-decoration: underline;">
                                     Show Details
                                 </span>
-                                <span class="loading-tools-in-progress" aria-label="Loading Details for {{testcase.name}}" hidden>
-                                    <i class="fas -notch fa-spin save-button"></i>Loading...
+                                <span class="loading-tools-in-progress" style="text-decoration: underline;" aria-label="Loading Details for {{testcase.name}}" hidden>
+                                    <i class="fas -notch fa-spin save-button"></i>Cancel Loading</u>
```

Comment:
I might have missed it but I don't see an opening `<u>` tag to match the closing tag here.  Is this an intentional change?

Reply:
This was an unintentional change.  The `<\u>` tag has been removed.